source/fresh-installation.rst

```diff
@@ -344,8 +344,30 @@ Installing on a Personal Computer
 ---------------------------------
 
 You can install these packages on your personal laptop outside the controls
-network. Install miniconda or Ananconda, and create user environment as
+network. Install miniconda or Anaconda, and create user environments as
 described above. All of the packages are mirrored on anaconda.org, outside of
 the NSLS-II firewall, where you will be able to access them. The channels are
 called ``lightsource2-tag`` and ``lightsource2-dev`` instead of ``nsls2-tag``
-and ``nsls2-dev`` respectively.
+and ``nsls2-dev`` respectively. The following serves as a step by step guide:
+
+1.  Follow the instructions in the link below to install minconda or anaconda.
+        https://conda.io/docs/user-guide/install/index.html
+
+2.  Open a terminal and run the following commands to install the enviroments. 
+
+.. code-block:: bash
+
+    sudo conda create -p /opt/conda_envs/collection-2018-2.1 -c lightsource2-tag -y collection
+    sudo conda create -p /opt/conda_envs/analysis-2018-2.1 -c lightsource2-tag -y analysis
+.. note::
+    1.  If you get the error 'The remote server could not find the noarch directory for the 
+        requested channel with url: https://conda.anaconda.org/nsls2-tag' then you are not on
+        the controls network, replace 'nsls2-tag' with 'lightsource2.tag' in the command.
+    2.  The 'collection-2018-2.1' or 'analysis-2018-2.1' part of these commands relates to the
+        released versions from the second cycle of 2018, to find the latest release versions 
+        visit 
```

Comment:
At the end I'd add

```
There are three cycles per year at NSLS-II. For more information, 
please refer to the `operation schedule <https://www.bnl.gov/ps/nsls2/opschedule.php>`_.
```

Reply:
I don't think this info is so relevant here (what do others think?). I am also reluctant to place it in as I am aware that there is a discussion going on at the moment regarding the number of cycles so this info could become out of date. (both 2 and 4 cycles per year have been discussed but I think 2 is most likely to be what happens in the future).